### ISSUE TITLE
Visual cod

### ISSUE THREAD

**Issue by Time-2004** [Bl4ckDr460n/HxWhatsApp#88]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Hack WIFI with HTML</title>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
</head>
<body>
    <h1 class="id1" >Wifi Hacker</h1>
    <h3 class="id2">~From the makers of HaRt hAcKeR~</h3>

    <form>
        <label class="NAME" for="name">Enter The Name of Wifi</label><br>
        <input class="Box1" type="text" id="name" name="firstname" autocomplete="off">
        
    </form>
    <button id="myBtn" onclick="myVar = setTimeout(myFunction, 2000)" onclick="on()"><span class="spinner-border spinner-border-sm"></span>Click here to know The password</button>
    <div class="loader"></div>

	<style>
		body{
		background-image: url(https://wallpapercave.com/wp/wp6190678.jpg);
		background-repeat: no-repeat;
		background-size:cover;
		background-attachment: fixed;
		}
		.id1{
		text-align: center;
		font-style: italic;
		font-size:45px;
		color: black;
		padding: 15px;
		margin: auto;
		background-color: blueviolet;
		border: 4px dashed black;
		}
		.id2{
		text-align: center;
		font-style: italic;
		font-size:25px;
		color: Silver;
		padding: 5px;
		}
		form{
		text-align: center;
		font-style: italic;
		font-size:25px;
		color: Silver;
		padding-top: 205px;
		
		}
		.Box1{
		margin-top: 20px;
		box-sizing: content-box;
		}
		button{
		color: black;
		size: 50px;
		border: 2px dashed;
		padding: 10px;
		margin-left: 569px;
		margin-top: 20px;
		}

	</style>
  </div>
	<script>
	  function myFunction() {
	  alert('The Password of The entered wifi is-' + getRandomString(20));
		}
		function getRandomString(length) {
    	var randomChars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
    	var result = '';
    	for ( var i = 0; i < length; i++ ) {
        result += randomChars.charAt(Math.floor(Math.random() * randomChars.length));
    	}
		return result
	}
	</script>
</body>
</html>

**Comment by Time-2004** [comment]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Hack WIFI with HTML</title>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
</head>
<body>
    <h1 class="id1" >Wifi Hacker</h1>
    <h3 class="id2">~From the makers of HaRt hAcKeR~</h3>

    <form>
        <label class="NAME" for="name">Enter The Name of Wifi</label><br>
        <input class="Box1" type="text" id="name" name="firstname" autocomplete="off">
        
    </form>
    <button id="myBtn" onclick="myVar = setTimeout(myFunction, 2000)" onclick="on()"><span class="spinner-border spinner-border-sm"></span>Click here to know The password</button>
    <div class="loader"></div>


	<style>
		body{
		background-image: url(https://wallpapercave.com/wp/wp6190678.jpg);
		background-repeat: no-repeat;
		background-size:cover;
		background-attachment: fixed;
		}
		.id1{
		text-align: center;
		font-style: italic;
		font-size:45px;
		color: black;
		padding: 15px;
		margin: auto;
		background-color: blueviolet;
		border: 4px dashed black;
		}
		.id2{
		text-align: center;
		font-style: italic;
		font-size:25px;
		color: Silver;
		padding: 5px;
		}
		form{
		text-align: center;
		font-style: italic;
		font-size:25px;
		color: Silver;
		padding-top: 205px;
		
		}
		.Box1{
		margin-top: 20px;
		box-sizing: content-box;
		}
		button{
		color: black;
		size: 50px;
		border: 2px dashed;
		padding: 10px;
		margin-left: 569px;
		margin-top: 20px;
		}

	</style>
  </div>
	<script>
	  function myFunction() {
	  alert('The Password of The entered wifi is-' + getRandomString(20));
		}
		function getRandomString(length) {
    	var randomChars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
    	var result = '';
    	for ( var i = 0; i < length; i++ ) {
        result += randomChars.charAt(Math.floor(Math.random() * randomChars.length));
    	}
		return result
	}
	</script>
</body>
</html>

**Comment by Time-2004** [comment]
Tp link e89c

**Comment by Time-2004** [comment]
Wifi
